Automattic/vip-go-mu-plugins
# Search: Remove call to get_last_sync() since it's being removed

## Description
In EP 4.0, the `get_last_sync()` util function is removed. Let's remove the check for it in `check_settings_health()` and `validate_contents()` since it's not a good indicator if the index exists. Rather, let's use `index_exists()` in `get_index_settings_health_for_all_indexables()`.

## Changelog Description

### Plugin Updated: Enterprise Search

Remove `get_last_sync()` from `check_settings_health()` and `validate_contents()` since that util function is deprecated in EP 4.0

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. `$es = new \Automattic\VIP\Search\Search();`
2. `$es->init();`
3. `$hj = new \Automattic\VIP\Search\SettingsHealthJob( $es );`
4. `$hj->health->get_index_settings_health_for_all_indexables();` — verify results are accurate

1. `$es = new \Automattic\VIP\Search\Search();`
2. `$es->init();`
3. `$health = new \Automattic\VIP\Search\Health( $es );`
4. `$health->validate_contents()`
